jpa/boot-data-multipledatasources/src/test/java/com/example/multipledatasources/service/impl/DetailsServiceImplTest.java

```diff
@@ -1,46 +0,0 @@
-package com.example.multipledatasources.service.impl;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.BDDMockito.given;
-
-import com.example.multipledatasources.dto.ResponseDto;
-import com.example.multipledatasources.entities.cardholder.CardHolder;
-import com.example.multipledatasources.entities.member.Member;
-import com.example.multipledatasources.repository.cardholder.CardHolderRepository;
-import com.example.multipledatasources.repository.member.MemberRepository;
-import java.util.Optional;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-
-@ExtendWith(MockitoExtension.class)
-class DetailsServiceImplTest {
-
-    @Mock
-    private CardHolderRepository cardHolderRepository;
-
-    @Mock
-    private MemberRepository memberRepository;
-
-    @InjectMocks
-    private DetailsServiceImpl detailsService;
-
-    @Test
-    void getDetails() {
-        // given
-        Member member = new Member();
-        member.setName("junit");
-        given(memberRepository.findByMemberId("1")).willReturn(Optional.of(member));
-
-        given(cardHolderRepository.findByMemberId("1")).willReturn(Optional.of(new CardHolder()));
```

Comment:
@coderabbitai, how to check for parallelization in service test as per change in main class